system/Session/Handlers/RedisHandler.php

```diff
@@ -204,19 +269,17 @@ public function close(): bool
     {
         if (isset($this->redis)) {
             try {
-                $pingReply = $this->redis->ping();
```

Comment:
I remove the `ping()` call, because when a ping is unsuccessful/redis connection is broken, phpredis will throw an exception, just as it will with `del()` below.